packages/angular/src/generators/convert-tslint-to-eslint/convert-tslint-to-eslint.ts

```diff
@@ -43,6 +43,12 @@ export async function conversionGenerator(
     },
   });
 
+  /**
+   * If root eslint configuration already exists it will not be recreated
+   * but we also don't want to re-run the tslint config conversion
+   * as it was likely already done
+   */
+  const rootEslintConfigExists = host.exists('.eslintrc.json');
```

Comment:
Is there any reason why we wouldn't want this same logic applied to all conversion? (i.e. cypress and nest currently)

If we just move it into the implementation details of `projectConverter.convertRootTSLintConfig` all conversion generators will get it for free (which was the intention behind sharing as much logic as possible)

Reply:
IMHO it make sense to move the entire sequence to a wrapper `convert` function in `projectConverter` which would include most of the duplicated `conversionGenerator` code:
* initialize esLint (with check for existing root `eslintrc`)
* convert root TSLint
* convert project TSLint
* remove project TSLint
* set defaults
* format
* uninstall tslint

Was there a reason of having it duplicated in the first place?